Tests iOS/ReaderTests.swift

```diff
@@ -55,6 +57,9 @@ class ReaderTests: XCTestCase {
     }
 
     override func tearDownWithError() throws {
+        Task {
```

Comment:
Wrapping this method in a Task will cause a race condition.

There's an async version of teardown you should use instead